app/presenters/related_items.rb

```diff
@@ -0,0 +1,37 @@
+class RelatedItems
+  attr_reader :contact
+
+  def initialize(contact)
+    @contact = contact
+  end
+
+  def payload
+    { sections: build_sections }
+  end
+
+private
+
+  def build_sections
+    sections = []
+
+    if contact.quick_links.to_a.any?
```

Comment:
why do you have to call `to_a`? on `app/views/contacts/show.html.erb:44` (deleted code) it was using `#any?` directly